projects/WeTek_Play/linux/linux.arm.conf

```diff
@@ -1043,20 +1062,20 @@ CONFIG_AM_WIFI_SD_MMC=y
 # CONFIG_DHD_USE_STATIC_BUF is not set
 CONFIG_AM_WIFI_USB=y
 # CONFIG_RTL8189ES is not set
-CONFIG_RTL8192EU=m
-CONFIG_RTL8723AU=m
+# CONFIG_RTL8192EU is not set
+# CONFIG_RTL8723AU is not set
 # CONFIG_RTL8723AS is not set
 # CONFIG_RTL8723BS is not set
-CONFIG_RTL8821AU=m
-CONFIG_RTL8812AU=m
-CONFIG_RTL8192CU=m
-CONFIG_RTL8192DU=m
-CONFIG_RTL8188EU=m
-CONFIG_RTL8188EU_MP=m
-CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=y
+# CONFIG_RTL8821AU is not set
+# CONFIG_RTL8812AU is not set
+# CONFIG_RTL8192CU is not set
+# CONFIG_RTL8192DU is not set
+# CONFIG_RTL8188EU is not set
+# CONFIG_RTL8188EU_MP is not set
+CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
```

Comment:
ah. when CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE is built-in it has no change to load the nvram config and firmware. unless it's not in initrd... maybe...